experiment_buddy/experiment_buddy.py

```diff
@@ -22,6 +22,7 @@
 import experiment_buddy.utils
 from experiment_buddy.utils import get_backend
 from experiment_buddy.utils import get_project_name
+from experiment_buddy.utils import is_running_remotely
```

Comment:
import experiment_buddy.utils

or if it's too long
from experiment_buddy import utils

Reply:
import! :D